x-pack/test/case_api_integration/spaces_only/tests/common/configure/post_configure.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import {
+  getConfigurationRequest,
+  removeServerGeneratedPropertiesFromSavedObject,
+  getConfigurationOutput,
+  deleteConfiguration,
+  createConfiguration,
+  getAuthWithSuperUser,
+} from '../../../../common/lib/utils';
+import { nullUser } from '../../../../common/lib/mock';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const authSpace1 = getAuthWithSuperUser();
+
+  describe('post_configure', () => {
+    afterEach(async () => {
+      await deleteConfiguration(es);
+    });
+
+    it('should create a configuration in space1', async () => {
```

Comment:
I think we should add a test where a user with permissions only to space 1 cannot create a configuration to space2

Reply:
So for spaces only (no security), is it possible to not have permissions to a space? I thought permissions weren't enforced when security was disabled.

Reply:
You are right! I was confused.